test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -145,6 +144,11 @@ public void TestJsonSchemaConfirmance(string description, string schemaText, str
             @"['warning','string-length-invalid','String 'str' length should be <= 2','file',1,13]
               ['warning','string-length-invalid','String 'str' length should be >= 4','file',1,13]")]
 
+        // string format validation
+        [InlineData("{'type': ['string'], 'format': 'date-time'}", "'1963-06-19T08:30:06Z'", "")]
+        [InlineData("{'type': ['string'], 'format': 'date-time'}", "'invalid'",
+            "['warning','format-invalid','String 'invalid' is not a valid 'DateTime'','file',1,9]")]
+
```

Comment:
can you add this test? `value<string>` might also throw

```csharp
[InlineData("{'type': ['string'], 'format': 'date-time'}", "1"]
```

Reply:
I also tried with
```
var v = new JValue(); // with all 13 overloads except JValue(JValue) & JValue(object)
var vStr = (string)v;
```
and they are all parsed with no exception.